### PR TITLE
fix typos in web3-eth-accounts.rst & TESTING.md #5047

### LINKED ISSUE #5047: Small spelling errors in web3-eth-accounts.rst & TESTING.md
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What would you like to say about Web3.js' documentation?

just some small typos

### Examples/References

double 'the', missing 'the' / 'to'

### PR DESCRIPTION
**Pull request by andreistefanwork**
## Description

Please include a summary of the changes and be sure to follow our [Contribution Guidelines](../blob/1.x/CONTRIBUTIONS.md).

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `npm run dtslint` with success and extended the tests and types if necessary.
- [ ] I ran `npm run test:cov` and my test cases cover all the lines and branches of the added code.
- [ ] I ran `npm run build` with success.
- [ ] I have tested the built `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [x] I have updated the `CHANGELOG.md` file in the root folder.

### CODE REVIEW

**Review by nazarhussain — APPROVED** [review]

**Comment by nazarhussain** [comment]
@andreistefanwork Please add an entry to the `CHANGELOG.md` referring the issue. Rest LGTM.

**Review by jdevcs — APPROVED** [review]

**Review by avkos — APPROVED** [review]

**Comment by andreistefanwork** [comment]
@nazarhussain Thanks for checking it out, entry added in `CHANGELOG.md`

File: CHANGELOG.md [hunk 1]
```diff
@@ -548,6 +548,7 @@ Released with 1.0.0-beta.37 code base.
 -  Fix static tuple encoding (#4673) (#4884)
 -  Fix bug in handleRevert logic for eth_sendRawTransaction (#4902)
 -  Fix resolve type of getBlock function (#4911)
+-  Fix typos in web3-eth-accounts.rst & TESTING.md (#5047)
```
Comment (jdevcs) [inline CHANGELOG.md]: It should be under 1.7.5 as 1.7.4 is already frozen. https://github.com/ChainSafe/web3.js/blob/release/1.7.4/CHANGELOG.md#174

Comment (jdevcs) [reply CHANGELOG.md]: @andreistefanwork

Comment (andreistefanwork) [reply CHANGELOG.md]: Correct .. should I add the 1.7.5 entry in `CHANGELOG.md`? (it's not in the file on 1.x currently)

**Review by luu-alex — APPROVED** [review]

**Pull request merged**